src/server_manager/web_app/app.ts

```diff
@@ -727,25 +714,11 @@ export class App {
     }
   }
 
-  // Opens the screen to create a server.
-  private showCreateServer() {
-    const regionPicker = this.appRoot.getAndShowRegionPicker();
-    // The region picker initially shows all options as disabled. Options are enabled by this code,
-    // after checking which regions are available.
-    this.digitalOceanRetry(() => {
-          return this.digitalOceanRepository.getRegionMap();
-        })
-        .then(
-            (map) => {
-              const locations = Object.entries(map).map(([cityId, regionIds]) => {
-                return this.createLocationModel(cityId, regionIds);
-              });
-              regionPicker.locations = locations;
-            },
-            (e) => {
-              console.error(`Failed to get list of available regions: ${e}`);
-              this.appRoot.showError(this.appRoot.localize('error-do-regions'));
-            });
+  private async showCreateServer() {
+    const digitalOceanCreateServer =
+        this.appRoot.getAndShowDigitalOceanCreateServer() as DigitalOceanCreateServer;
```

Comment:
Do you need the "as"?
Perhaps put the type in the Javascript code? That function should be properly typed.